sda-download/api/sda/sda.go

```diff
@@ -272,23 +272,18 @@ func Download(c *gin.Context) {
 		return
 	}
 
-	switch c.Param("type") {
-	case "encrypted":
-		// calculate coordinates
-		start, end, err = calculateEncryptedCoords(start, end, c.GetHeader("Range"), fileDetails)
-		if err != nil {
-			log.Errorf("Byte range coordinates invalid! %v", err)
+	wholeFile := true
+	if start != 0 || end != 0 {
+		wholeFile = false
+	}
```

Comment:
is the condition 
```golang
start == 0 && end >= fileDetails.DecryptedSize
```
also satisfy the condition `wholeFile`? 
so maybe we can change the code to 
```suggestion
        wholeFile := false
        if start == 0 && (end == 0 || end >= fileDetails.DecryptedSize) {
              wholeFile = true
        }
```

Reply:
make sense to me.